WalletWasabi.Fluent/ViewModels/Wallets/Home/Tiles/PrivacyRing/PrivacyBarItemViewModel.cs

```diff
@@ -1,28 +1,53 @@
 using Avalonia;
 using Avalonia.Media;
+using System.Linq;
 using WalletWasabi.Blockchain.TransactionOutputs;
 using WalletWasabi.Fluent.Helpers;
+using WalletWasabi.Fluent.Models;
 using WalletWasabi.Fluent.ViewModels.Wallets.Advanced.WalletCoins;
+using WalletWasabi.Wallets;
 
 namespace WalletWasabi.Fluent.ViewModels.Wallets.Home.Tiles.PrivacyRing;
 
-public class PrivacyBarItemViewModel : WalletCoinViewModel
+public class PrivacyBarItemViewModel : ViewModelBase, IDisposable
 {
-	public PrivacyBarItemViewModel(PrivacyBarViewModel parent, SmartCoin coin, double start, double width) : base(coin)
+	public const double BarHeight = 10;
+
+	public PrivacyBarItemViewModel(PrivacyBarViewModel parent, SmartCoin coin, double start, double width)
 	{
+		Coin = new WalletCoinViewModel(coin);
+
 		IsPrivate = coin.IsPrivate(parent.Wallet.KeyManager.AnonScoreTarget);
 		IsSemiPrivate = !IsPrivate && coin.IsSemiPrivate();
```

Comment:
It is actually a bug, probably introduced by me.
`IsSemiPrivate()` Should receive the `AnonScoreTarget` as param too. Anyway, not related to this PR.